sdcm/nemesis.py

```diff
@@ -832,6 +832,26 @@ def disrupt_snapshot_operations(self):
         if result.stderr:
             self.tester.fail(result.stderr)
 
+    def disrupt_alter_encryption(self, keyspace="keyspace1", table="standard1"):
```

Comment:
Need to change the nemesis name to enable/disable encryption or something like that.
To differ it from additional nemesis that "alters" encryption properties.